docs/design_docs/20230405-default_value.md

```diff
@@ -0,0 +1,80 @@
+# MEP: Default Value
+
+Current state: Under Discussion
+
+ISSUE: [[Feature]: Support Default Value #23337](https://github.com/milvus-io/milvus/issues/23337)
+
+Keywords: Default, Insert, Upsert 
+
+Released: v2.4.0
+
+## Summary
+
+Support Default Value when input data.
+
+## Motivation
+
+For now, Milvus don't support Default function. If the user pass in the same data under a certain field schema, the data can only be passed in repeatedly, which is not so flexible and user-friendly。
+
+We need a way to support Default function, which is more efficient.
+
+## Public Interfaces
+
+Add new field `default_value` in `FieldSchema`
+```proto
+message FieldSchema {
+  ...
+  repeated schema.FieldData default_value = 10;
+}
+```
+
+## Design Details
+
+1. Add the default_value in the field schema as an optional field.
+
+```proto
+message FieldSchema {
+  ...
+  repeated schema.FieldData default_value = 10;
```

Comment:
why shall `default_value` be repeated here?

Reply:
Sorry, it's a mistake, I will correct it.